pipeline_steps/pubcloud.groovy

```diff
@@ -237,8 +237,11 @@ def runonpubcloud(Map args=[:], Closure body){
 }
 
 def uploadToSwift(Map args){
```

Comment:
I would prefer to either have the function take an artefacts directory argument or the function to fail if `env.RE_HOOK_ARTIFACT_DIR` isn't set. I think having a default here creates a potential bug.